test/scenarios/12. decrypt/encrypt and decrypt.csv

```diff
@@ -0,0 +1,15 @@
+Steps,Test Steps,Test Data,Expected Result,Notes
+1,Open the extension.,,The Welcome Back screen is shown.,
+2,Proceed to Unlock the wallet.,password (8 characters min).,"The Ether balance is shown on the overview.
+The wallet address is shown on the overview.
+The selected network is Ethereum Mainnet.",
+3,Switch networks to a test network.,e.g. Goerli.,"The Goerli balance is shown on the overview.
+The wallet address is shown on the overview.
+The selected network is Goerli.",
+4,Open the test dapp in another tab.,https://metamask.github.io/test-dapp/,,
+5,Proceed to Connect with MetaMask.,,,
+6,Connect with the current account.,,,
+7,Get encryption key.,,The message is in plain text format.,
+8,Encrypt message.,,,
```

Comment:
Can we add the step for adding a message to encrypt

Reply:
Done, thanks!